index.html

```diff
@@ -0,0 +1,490 @@
+                                            <div class="m-auto creator-name-subs">
+                                                <span class="">Anikriz Media</span> <br>
+                                                <span style="font-size: 10px;" class="">500 subscibers</span>
+                                            </div>
+                                        </div>
+                                        <div class="creator-right col-6">
+                                            <div class="creator-inner">
+                                                <button type="button" class="btn btn-danger">SUBSCRIBE</button>
+                                                <i data-feather="bell"></i>
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div class="d-flex justify-content-between col-md-12 col-sm-12 col-lg-12 border border-secondary one-of-two-box p-1">
+                                        <div class="col-4 comment-and-likes m-auto">
+                                            <span class="">Comments</span> <br>
+                                            <span style="font-size: 10px;" class="">1.1K</span>
+                                        </div>
+                                        <div class="col-8 d-flex justify-content-start">
+                                            <img src="img/trey-songz.jpg" alt="Trey Songz" class="rounded-circle mt-2" style="width: 25px; height: 25px;">
+                                            <div class="ms-2 comment-text">
+                                                <span style="font-size: 10px;" class="">You asked me for a simple strategy, so here it is. I ...</span>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </section>
+                    <section>
+                        <div class="comment-section">
+                            <div class="comment-no-and-sorts d-flex justify-content-start">
+                                <div class="me-5">1,123 Comments</div>
+                                <div class="d-flex justify-content-start">
+                                    <i data-feather="align-left"></i>
+                                    <p class="ms-2">SORT BY</p>
+                                </div>
+                            </div>
+                            <div class="form-section d-flex justify-content-start">
+                                <div class="commenter-image col-1">
+                                    <img src="img/trey-songz.jpg" alt="Trey Songz" class="rounded-circle m-auto" style="width: 40px; height: 40px;">
```

Comment:
Please do not use inline styles. Keep your style definitions in the external CSS file you created.